File: flutter-main/flutter-main/Pattern2/p6.dart
import 'dart:io';
void main () {
        int num=4;
        int row=num;
        int dec=2;
        for (int i=1; i<=num; i++, row--) {
                for(int j=0; j<row; j++,dec=dec+2) {
                        stdout.write ("${dec}   ");

                }
                print("");
        }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session5/p6.c
//           1
//      4    7
//10   13   16

#include<stdio.h>
void main(){
	int row;
	printf("enter the row:\n");
	scanf("%d",&row);

	int num=1;
	for(int i=0;i<=row;i++){

		for(int sp=row;sp>i;sp--){
			printf(" \t");
		}
		for(int j=1;j<=i;j++){
			printf("%d\t",num);
			num=num+row;
		}
		printf("\n");
	}
}


